src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -1963,71 +2365,6 @@ private void EncryptTransaction(byte* fullTxBuffer)
                 throw new InvalidOperationException("Failed to call crypto_aead_xchacha20poly1305_ietf_encrypt, rc = " + rc);
         }
 
-        private sealed class CompressionAccelerationStats
```

Comment:
Is it something no longer valuable for shared journals only or we should consider removing it in 7.0 as well?